tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/optimizer/GenIncOptimizer.scala

```diff
@@ -22,11 +22,12 @@ import Trees._
 import Types._
 
 import org.scalajs.core.tools.sem._
-import org.scalajs.core.tools.javascript.{ESLevel, LongImpl}
+import org.scalajs.core.tools.javascript.ESLevel
 import org.scalajs.core.tools.logging._
 
 import org.scalajs.core.tools.linker._
 import org.scalajs.core.tools.linker.analyzer.SymbolRequirement
+import org.scalajs.core.tools.linker.backend.emitter.LongImpl
```

Comment:
This makes a nice caveat apparent: The optimizer is not agnostic to the backend used because of longs :)